checker/src/main/java/org/checkerframework/checker/nonempty/NonEmptyTransfer.java

```diff
@@ -370,4 +393,20 @@ private Integer emptyValue(Node n) {
             return null;
         }
     }
+
+    /**
+     * If the transfer result given is regular result, then turn it into a conditional result while
+     * preserving the store. Otherwise, do nothing.
+     *
+     * @param result an input transfer result for regularity check
+     * @return a conditional transfer result
+     */
+    private TransferResult<CFValue, CFStore> regularCheck(TransferResult<CFValue, CFStore> result) {
```

Comment:
"Check" sounds like nothing is happening and I would have expected a boolean return type. Can you think of a better name? Maybe `ensureConditionalTransferResult`?